pkg/commands/git_commands/config.go

```diff
@@ -111,3 +111,7 @@ func (self *ConfigCommands) GetCoreCommentChar() byte {
 func (self *ConfigCommands) GetRebaseUpdateRefs() bool {
 	return self.gitConfig.GetBool("rebase.updateRefs")
 }
+
+func (cfgCommands *ConfigCommands) GetShowNumberOfLineChanges() bool {
```

Comment:
This file is for accessing values from the git config, as opposed to the user config. I would instead just use `.UserConfig().Gui.ShowNumberOfLineChanges` directly wherever you need it